CONTRIBUTING.rst

```diff
@@ -236,3 +236,15 @@ but here is a simple overview:
     base: features        # if it's a feature
 
 
+Joining the Development Team
+----------------------------
+
+Anyone who has successfully seen through a pull request which did not
+require any extra work from the development team to merge will
+themselves gain commit access (if we forget please send a friendly
+reminder).  This does not mean your workflow to contribute changes,
```

Comment:
I had to re-read this a couple of times, parsed it as "(This does not mean your workflow to) (contribute changes)" (contributing changes, not changing a workflow) :wink: